configure.ac

```diff
@@ -3315,6 +3317,8 @@ case "$OPT_H2" in
 esac
 
 curl_h2_msg="disabled (--with-nghttp2)"
+AM_CONDITIONAL([USE_CPPFLAG_NGHTTP2_STATICLIB],
+  [test "x$xc_lt_build_static_only" = 'xyes'])
```

Comment:
Here too, this seems to use the request to build a static-only version of curl to suggest that nghttp2 is also static, which it doesn't say!